lcc_members/models/res_partner.py

```diff
@@ -333,6 +333,7 @@ def create_public_profile(self):
                     "type": "other",
                     "contact_id": partner.id,
                     "partner_profile": profile["id"],
+                    "company_id": partner.company_id.id,
```

Comment:
Think  that not true .. link between company and contact is un position_profile .. public profile is unique and one personne can have many positions and can belongs to many compagny